frontend/public/components/utils/_file-input.scss

```diff
@@ -52,3 +52,7 @@
   top: 50%;
   transform: translate(-50%, -50%);
 }
+
+.co-file-dropzone__file-upload-error {
+  color: $color-red-error;
```

Comment:
Should we use the Bootstrap class `has-error` instead? Then we don't need the custom style